.github/workflows/build_micromamba.yml

```diff
@@ -264,12 +264,12 @@ jobs:
           ls -al
           echo $(pwd)
           TMP_MAMBA_EXE="${{ steps.get_micromamba.outputs.micromamba_bin_linux_64 }}"
-          cp $TMP_MAMBA_EXE ./micromamba
-          chmod 775 ./micromamba
+          mkdir -p $HOME/mbins
+          cp $TMP_MAMBA_EXE $HOME/mbins
```

Comment:
this is wrong. 

You need the full path: cp $TMP_MAMBA_EXE $HOME/mbins/micromamba as it was before, to change the name of the final binary to `micromamba` from `micromamba-linux-64`.